src/read_vegparam.c

```diff
@@ -239,16 +239,23 @@ veg_con_struct *read_vegparam(FILE *vegparam,
         nrerror(ErrStr);
       }
 
+      for ( j = 0; j < 12; j++ ) {
```

Comment:
Its just good practice and we do merge this back into VIC 5, even if we wont be keeping it for long.  So if we don't do it now, I always have to clean up these bug fixes when they go into develop.